hazelcast-client-legacy/src/test/java/com/hazelcast/client/ClientRegressionWithMockNetworkTest.java

```diff
@@ -616,17 +615,23 @@ public void entryAdded(EntryEvent<Object, Object> event) {
     public void testDeadlock_WhenDoingOperationFromLifecycleListener() {
```

Comment:
Ok, added a it as a new test with name testDeadlock_WhenDoingOperationFromLifecycleListenerWithInitialPartitionTable.